app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :likes
```

Comment:
- [ ] It is important we follow the ERD diagram, to use the customized `author_id` field from the user model, here is what we can do. For example; Relationship between a User and a Post model,
```
# app/models/user.rb
has_many :posts, foreign_key: 'author_id'

# app/models/post.rb
belongs_to :author, class_name: 'User'

```
Let us do this for all other models that share the relationship :+1: